src/components/RevokeContainer/RevokedScreen/RevokedScreen.test.tsx

```diff
@@ -0,0 +1,203 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { RevokedScreen } from "./RevokedScreen";
+import { useRevokeQueue } from "../../../common/hook/useRevokeQueue";
+import { RevokeState } from "../../../constants/RevokeState";
+import sampleConfig from "../../../test/fixtures/sample-config-ropsten.json";
+import sampleRevokableDocument from "../../../test/fixtures/sample-wrapped-document.json";
+import { Config } from "../../../types";
+import { getDefaultProvider, Wallet } from "ethers";
+
+const config = {
+  ...sampleConfig,
+  wallet: Wallet.createRandom().connect(getDefaultProvider("ropsten")),
+} as Config;
+
+jest.mock("../../../common/hook/useRevokeQueue");
+
+const mockUseRevokeQueue = useRevokeQueue as jest.Mock;
+const mockRevoke = jest.fn();
+
+describe("RevokedScreen", () => {
+  it("should render the correct title based when revoke state is INITIALIZED", () => {
+    mockUseRevokeQueue.mockReturnValue({
+      revoke: mockRevoke,
+      revokeState: RevokeState.INITIALIZED,
+      revokedDocuments: [sampleRevokableDocument],
+      failedRevokedDocuments: [],
+      pendingRevokeDocuments: [],
+    });
+    render(
+      <RevokedScreen
+        config={config}
+        revokeDocuments={[sampleRevokableDocument]}
+        revokeAnotherDocument={() => {}}
+        onLogout={() => {}}
+        fileName="doc-1.tt"
+      />
+    );
+    expect(screen.queryByTestId("title")?.textContent).toStrictEqual("Please wait while we prepare your document(s)");
+  });
```

Comment:
doesn't sound right in relationship to test statement

Reply:
what do u mean? when state is initialised, it will be preparing the document into jobs.